content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -181,7 +181,7 @@ However, the particular path specified in the custom recycler pod template in th
 
 {{< feature-state for_k8s_version="v1.11" state="beta" >}}
 
-Support for expanding PersistentVolumeClaims (PVCs) is now enabled by default. You can expand
+Support for expanding *persistent volume claims* (PVCs) is now enabled by default. You can expand
```

Comment:
`PersistentVolumeClaims`?

Reply:
There is no object called "PersistentVolumeClaims".